pkg/api/devops/v1alpha3/pipeline_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The KubeSphere Authors.
+Copyright 2022 The KubeSphere Authors.
```

Comment:
If I understand it correctly, we cannot just change it directly. See also https://opensource.stackexchange.com/questions/6389/how-do-the-years-specified-in-a-copyright-statement-work

Reply:
IMO, if we never change this file, we'd better not modify it directly.

However, we have noticed https://github.com/apache/skywalking-eyes recently which [Skywalking](https://github.com/apache/skywalking/blob/master/.licenserc.yaml) is using for checking LICENSE header. I'm wondering if we can use it to reduce uncertainty.

Reply:
Thanks for your proposal, please feel free to file an issue about it.